src/scenes/home/signup/signup.js

```diff
@@ -134,7 +138,7 @@ class SignUp extends Component {
             <li className={styles.errorMessage}>{this.state.error}</li>
           </ul> : null }
           {this.state.success && <Redirect to="/thanks" />}
-          <FormButton className={styles.joinButton} text="Join" onClick={this.handleOnClick} theme="red" />
+          {this.state.isLoading ? <FormButton className={styles.joinButton} text="Loading..." disabled={this.state.isLoading} theme="red" /> : <FormButton className={styles.joinButton} text="Join" onClick={this.handleOnClick} theme="red" />}
```

Comment:
On this line, you can just do `disabled={true}` since we already have checked that `isLoading` is true - small performance boost!

So, you're idea is correct, but the `<FormButton>` component doesn't currently support disabled. I went ahead and added that functionality for you. I'm just gonna copy-paste it cuz I didn't wanna step on your contribution!

```
import React, { Component } from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import styles from './formButton.css';

class FormButton extends Component {
  render() {
    const classes = classNames({
      [`${styles.formButton}`]: true,
      [`${styles[this.props.theme]}`]: true,
      [`${this.props.className}`]: this.props.className
    });
    return (
      <button
        type="button"
        className={classes}
        onClick={this.props.onClick}
        disabled={this.props.disabled}
      >
        {this.props.text}
      </button>
    );
  }
}

FormButton.propTypes = {
  theme: PropTypes.string,
  text: PropTypes.string.isRequired,
  onClick: PropTypes.func,
  className: PropTypes.string,
  disabled: PropTypes.boolean,
};

FormButton.defaultProps = {
  theme: 'blue',
  onClick: null,
  className: null,
  disabled: false,
};

export default FormButton;

```

So feel free to make those changes. Additionally, you should try to add these 2 features:
1. Change the button color to grey or a greyer red if `props.disabled` is true.
2. Add an optional prop to the `<FormButton>` component for choosing the cursor type. I think this one should be `wait` on hover, but there may exist a `<FormButton>` in the future where we'd like to do a `not-allowed` on hover.